evennia/typeclasses/attributes.py

```diff
@@ -414,6 +470,366 @@ def reset_cache(self):
         self._cache = {}
         self._catcache = {}
 
+    def create_attr(self, keystr, category, lockstring, value):
```

Comment:
Further on there is some inconsistencies in the naming convention here, with a mixture of `_attr` and `_attributes`. I think it's clearer if you name all these Backend methods with the full `_attribute(s)` (so this would be `create_attribute`) since these are already not going to be named after the corresponding AttributeHandler methods.